src/main/java/dk/cngroup/lentils/controller/TeamController.java

```diff
@@ -77,7 +90,7 @@ public String delete(@PathVariable("id") final Long id) {
     private void fillModelAttributes(
             final Model model,
             final List<Team> teams,
-            final Team team,
+            final TeamDTO team,
```

Comment:
V ramci dodrzeni konvence bych to prejmenoval taky na `teamDto` (vzhledem k tou ze na radku 70 mas taky `TeamDTO teamDto`).